docs/mounting.md

```diff
@@ -141,50 +141,51 @@ If the user mounting the Cloud Storage FUSE is not root then the ```allow_other`
-|--retry-multiplier value|Param for exponential backoff algorithm, which is used to increase waiting time b/w two consecutive retries. (default: 2)|
-|--experimental-local-file-cache|Experimental: Cache GCS files on local disk for reads.|
-|--temp-dir value|Path to the temporary directory where writes are staged prior to upload to Cloud Storage. (default: system default, likely /tmp)|
-|--client-protocol value|The protocol used for communicating with the GCS backend. Value can be 'http1' (HTTP/1.1) or 'http2' (HTTP/2). (default: http1)|
-|--max-conns-per-host value|The max number of TCP connections allowed per server. This is effective when --client-protocol is set to 'http1'. (default: 100)|
-|--max-idle-conns-per-host value|The number of maximum idle connections allowed per server (default: 100)|
-|--enable-nonexistent-type-cache|Once set, if an inode is not found in GCS, a type cache entry with type NonexistentType will be created. This also means new file/dir created might not be seen. For example, if this flag is set, and flag type-cache-ttl is set to 10 minutes, then if we create the same file/node in the meantime using the same mount, since we are not refreshing the cache, it will still return nil.|
-|--stackdriver-export-interval value|Experimental: Export metrics to stackdriver with this interval. The default value 0 indicates no exporting. (default: 0s)|
-|--experimental-opentelemetry-collector-address value|Experimental: Export metrics to the OpenTelemetry collector at this address.|
-|--log-file value|The file for storing logs that can be parsed by fluentd. When not provided, plain text logs are printed to stdout.|
-|--log-format value|The format of the log file: 'text' or 'json'. (default: "json")|
-|--debug_fuse_errors|If false, fuse errors will not be logged to the console (in case of --foreground) or the log-file (if specified)|
-|--debug_fuse|Enable fuse-related debugging output.|
-|--debug_fs|Enable file system debugging output.|
-|--debug_gcs|Print GCS request and timing information.|
-|--debug_http| Dump HTTP requests and responses to/from GCS, doesn't work when --enable-storage-client-library flag is true.|
-|--debug_invariants|Panic when internal invariants are violated.|
-|--debug_mutex|Print debug messages when a mutex is held too long.|
-|--enable-storage-client-library|If true, will use go storage client library otherwise jacobsa/gcloud|
-|--help, -h|Show help|
-|--version, -v|Print version|
+| Option                                               | Description                                                                                                                                                                                                                                                                                                                                                                                  | 
```

Comment:
For reviewer: After adding --http-client-timeout flag in the doc and saving the doc, these formatting changes happened automatically. You can open the readme doc in new tab and verify.

Reply:
Try editing by removing the last `|` delimiter and add `|` back in the last after the complete change. It worked for me.